src/php/DataSift/Stone/ComparisonLib/ComparitorBase.php

```diff
@@ -171,14 +176,7 @@ public function equals($expected)
 
 		// diff the two files to see what has changed
 		$differences = trim(`diff -ud --label=EXPECTED $tmpNam1 --label=ACTUAL $tmpNam2`);
-
-		// how did we do?
-		if (empty($differences)) {
-			$result->setHasPassed();
```

Comment:
Isn't this still needed if the `diff` matches?

Reply:
If `$sourceA` is the same as `$sourceB`, then it will return from the function early, so it will not reach this part.